addons/self-referencing/userscript.js

```diff
@@ -0,0 +1,26 @@
+export default async function ({ addon, global, console, msg }) {
+  const Blockly = await addon.tab.traps.getBlockly();
+  const vm = addon.tab.traps.vm;
```

Comment:
```suggestion
  const vm = await addon.tab.traps.vm;
```
You want to `await` the VM -- otherwise users will experience race-condition-related errors.

Reply:
@lisa-wolfgang Whoops, this is not correct. VM is synchronous in SA.
You won't find any SA API where you have to await a getter. SA APIs only return promises when calling some functions